docs/CHANGELOG_UI_CALLING.md

```diff
@@ -1,9 +1,22 @@
 # Azure Communication UI Calling Release History
 
-## 1.11.0-beta.1 (*)
+## 1.11.0-beta.2 (*)
+
+### Features
+- Call screen information header title/subtitle customization
+- Call screen control bar custom button support API change
```

Comment:
It's not only call screen. Setup screen as well
`Ability to hide or disable buttons and create custom buttons.`